.gitignore

```diff
@@ -10,3 +10,5 @@ __dummy.html
 
 # library
 *.a
+
+tags
```

Comment:
This probably belongs in your personal global gitignore, not the project gitignore